### PR TITLE
Social icons

### LINKED ISSUE #554: [add icon] a linkedin icon can be added
### Description

Instead of an arrow on **Follow me on Linkedin** adding an icon there will be better and aesthetically cool.

### Screenshots

image

### Additional information

for designers a Behance or dribble option adding will be good

### PR DESCRIPTION
**Pull request by mohammad4kh**
Adding different social media icons to the site

For example

 - Facebook
 - Instagram
 - Linkedin
 - Vimeo
 - Discord

and others 

Screenshot from 2021-10-01 02-33-57

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/366"></a>

Resolve #554

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Looks great 👍 Thank you!

Could you resolve the conflict please

**Review by AbhijithGanesh — APPROVED** [review]

**Comment by mohammad4kh** [comment]
@eddiejaoude thank you
the conflict is now resolved.

File: src/index.css [hunk 1]
```diff
@@ -11,3 +11,33 @@ code {
...
+  -webkit-box-shadow: 0px 8px 20px -9px #000000;
+  box-shadow: 0px 8px 20px -9px #000000;
+  transform: translateY(-5px);
+}
+
+.p-button {
+  transition: transform 0.33s !important;
+}
+
+.p-button:hover {
```
Comment (eddiejaoude) [inline src/index.css:41]: please can you check for existing/duplicate styles https://github.com/EddieHubCommunity/LinkFree/blob/main/src/Components/Links.css#L5

**Review by krishguptadev — CHANGES_REQUESTED** [review]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please take a look at the old requested changes.

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please see comments above

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Comment by krishguptadev** [comment]
Hey there! Just checking in 👋 Do you need any help?

**Comment by emmalearnscode** [comment]
As there are conflicts and this PR hasn't been worked on for a long time I will close it for now. Please feel free to open it if you get the time to fix the conflicts and make the requested updates. Thanks for contributing and please feel free to come back and help out again in the future!

**Pull request closed without merging**